### PR TITLE
Ss/removing psutil

### PR DESCRIPTION
**Pull request by SiddhantSadangi**

### CODE REVIEW

File: integrations-and-supported-tools/colab/Neptune_Colab.ipynb [hunk 1]
```diff
@@ -341,86 +341,85 @@
...
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "id": "Zgh3NwDoJAuG"
+    "id": "_r0Er7EHHram"
    },
    "outputs": [],
    "source": [
-    "run.stop()"
+    "run.print_structure()"
```
Comment (kamil-kaczmarek) [inline integrations-and-supported-tools/colab/Neptune_Colab.ipynb:422]: You cannot call this on the closed run. It should probably be above the `run.stop()`.

Comment (kamil-kaczmarek) [reply integrations-and-supported-tools/colab/Neptune_Colab.ipynb:422]: This will cause tests to fail.

**Review by kamil-kaczmarek — CHANGES_REQUESTED** [review]

**Comment by kamil-kaczmarek** [comment]
@SiddhantSadangi there is only one thing to change and we can mege it :)

File: integrations-and-supported-tools/colab/Neptune_Colab.ipynb (hunk 1 above)
Comment (SiddhantSadangi) [reply integrations-and-supported-tools/colab/Neptune_Colab.ipynb]: fixed

**Pull request merged**